resources/public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,10973 @@
+            examples:
+              example-1:
+                value:
+                  accountId: '8765432100'
+        required: true
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/lifetimeIsaManagerAccountsReinstateAccountResponse'
+              example:
+                status: 200
+                success: true
+                data:
+                  message: This account has been reinstated
+                  accountId: '8765432100'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/lifetimeIsaErrorResponse'
+              examples:
+                invalidPayload:
+                  value:
+                    code: BAD_REQUEST
+                    message: Bad Request
+                    errors:
+                    - code: INVALID_FORMAT
+                      message: Invalid format has been used
+                      path: /accountId
+                invalidLmrn:
+                  $ref: '#/components/examples/invalidLmrn'
+        '401':
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/lifetimeIsaErrorResponse'
+              examples:
+                notOpenStatus:
+                  $ref: '#/components/examples/notOpenStatusUnauthorized'
+        '404':
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/lifetimeIsaErrorResponse'
+              examples:
+                accountIdNotFound:
+                  $ref: '#/components/examples/accountIdNotFound'
+        '403':
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/lifetimeIsaErrorResponse'
+              examples:
+                accountClosed:
+                  value:
+                    code: INVESTOR_ACCOUNT_ALREADY_CLOSED
+                    message: You cannot reinstate this account because it was closed with a closure reason of transferred out
+                accountClosedCancelled:
+                  value:
+                    code: INVESTOR_ACCOUNT_ALREADY_CANCELLED
+                    message: You cannot reinstate this account because it was closed with a closure reason of cancellation
+                accountVoided:
+                  value:
+                    code: INVESTOR_ACCOUNT_ALREADY_OPEN
+                    message: You cannot reinstate this account because it is already open
+                complianceFail:
```

Comment:
This is missing a description. Please add, thanks.

                complianceFail:
                  description: You cannot reinstate this account because the investor has failed a compliance check
                  value:
                    code: INVESTOR_COMPLIANCE_CHECK_FAILED
                    message: You cannot reinstate this account because the investor has failed a compliance check